packages/components-js/tests/e2e/specs/dsr-ponyfill.e2e.ts

```diff
@@ -74,7 +74,9 @@ const initPorscheDesignSystem = async (page: Page): Promise<void> => {
   expect(amount).toBe(2);
 };
 
-test('should handle initial phase correctly', async ({ page }) => {
+test('should handle initial phase correctly', async ({ page, browserName }) => {
+  // firefox seams to boostrap declarative shadow dom differently in playwright than in real browser
+  test.fixme(browserName === 'firefox');
```

Comment:
Use "skipInBrowsers" helper for consistency, you could also adjust it so it uses "fixme" instead of "skip"

Reply:
`skipInBrowsers()` helper is a consciously decision to always skip it in the browser. But in this case it should be fixed since it should work in FF too.